test/spec/task/task.spec.ts

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import express from 'express';
+import sinon from 'sinon';
+import TaskServiceManager from '../../../src/modules/task/task-service-manager';
+import bodyParser from 'body-parser';
+import AccesstokenServiceManager from '../../../src/modules/access-token/access-token-manager';
+import ConfigService from '../../../src/modules/config/config-service';
+import TaskService from '../../../src/modules/task/task-service';
+
+chai.use(chaiHttp);
+
+let sinonSandbox: sinon.SinonSandbox;
+let appOne: any;
+let appTwo: any;
+let accessToken: string;
+const accountCreds: any = {
+  username: 'Sample Username',
+  password: 'Sample Password',
+};
+let accountId: string;
+
+describe.skip('Task Service.', () => {
+  before(async () => {
+    
+    appOne = express();
+    appTwo = express();
+
+    appOne.use(bodyParser.urlencoded({ extended: true }));
+    appOne.use(bodyParser.json());
+    appTwo.use(bodyParser.urlencoded({ extended: true }));
+    appTwo.use(bodyParser.json());
+    
+    appOne.use('/', await AccesstokenServiceManager.createRestAPIServer());
+    appTwo.use('/', await TaskServiceManager.createRestAPIServer());
+
+  });
+
+  beforeEach(async () => {
+    sinonSandbox = sinon.createSandbox();
+    sinonSandbox.stub(ConfigService, 'getStringValue').returns('1h');
+    const res = await chai
+      .request(appOne)
+      .post('/api/access-tokens')
+      .set('content-type', 'application/json')
+      .send(accountCreds);
+    accessToken = res.body.token;
+    accountId = res.body.accountId;
+  });
+
+  afterEach(() => {
+    sinonSandbox.restore();
+  });
+
+  it('GET "/api/accounts/:accountId/tasks" should return a list of all tasks for a particular accountId.', async () => {
+
+    // creating some tasks
+    for(let i = 0; i < 2;i++) {
+      await chai
+        .request(appTwo)
+        .post(`/api/accounts/${accountId}/tasks`)
+        .set('content-type', 'application/json')
+        .set('Authorization', `Bearer ${accessToken}`)
+        .send({
+          name: `${i}`
+        });
+    }
+
+    // getting a list of tasks
+    const res = await chai
+      .request(appTwo)
+      .get(`/api/accounts/${accountId}/tasks`)
+      .set('content-type', 'application/json')
+      .set('Authorization', `Bearer ${accessToken}`)
+      .send();
+
+    expect(res).to.have.status(200);
+    expect(res.body).to.be.an('array');
+
+    // As we are not mocking DB and the tests are running on real DB URL. 
```

Comment:
Can we clean up the database entry or create a new account before running this spec?  I think we should check how many are associated with this account based on how many we created at the beginning of the spec